### ISSUE TITLE
[Bug]: Stacked bee jars reset the the jar when placing a bee

### ISSUE THREAD

**Issue by EnderShadow** [Resourceful-Bees/ResourcefulBees#273]
### Bug Description

If you have a stack of bee jars filled with a bee that has a size larger than 1, placing the bee will set the entire stack of jars to being empty. While this normally cannot occur from finding wild bees or breeding them, it does occur when pollination spawns a filed bee jar as an item.

### How to Reproduce?

1. Have a bee pollinate a block such that the block mutates into a filled bee jar.
2. Repeat until you have more than 1 bee in a stack
3. Place the bee in the world from the stack.

### Expected Behavior

The stack of bee jars should decrease by one and the empty bee jar should either be added to the players inventory or dropped on the ground.

### Version

0.9.9.1

### Mod Loader Version

1.16.5 - 36.2.2

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by Epic428** [comment]
Go in creative and middle click a jar in jei with cheat mode activated. You should get a stack of bee jars. Then go into survival and place one and tell me if the same thing happens…

**Comment by Vaerys-Dawn** [comment]
it does, the bee jar code doesn't take into account how many are in the stack as usually jars don't stack because of extra nbt

**Comment by Vaerys-Dawn** [comment]
it should be an easy fix

**Comment by Epic428** [comment]
It never did that before….

**Comment by Epic428** [comment]
I grab stacks in creative all the time

**Comment by Vaerys-Dawn** [comment]
bee jars full of bees? or just bee jars

**Comment by Vaerys-Dawn** [comment]
before:
```java
stack.setTag(null);
```

after
```Java
if (stack.getCount() > 1) {
    if (!player.addItem(new ItemStack(ModItems.BEE_JAR.get()))) {
        player.drop(new ItemStack(ModItems.BEE_JAR.get()), false);
    }
    stack.shrink(1);
} else {
    stack.setTag(null);
}
```

**Comment by Vaerys-Dawn** [comment]
fixed in 0.9.9.4
